ui/components/src/model/units/percent.ts

```diff
@@ -0,0 +1,53 @@
+// Copyright 2022 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { UnitGroupConfig, UnitConfig } from './types';
+import { DEFAULT_DECIMAL_PLACES } from './constants';
+
+const percentUnitKinds = ['Percent', 'PercentDecimal', '%'] as const;
+type PercentUnitKind = typeof percentUnitKinds[number];
+export type PercentUnitOptions = {
+  kind: PercentUnitKind;
+  decimal_places?: number;
+};
+export const DECIMAL_GROUP_CONFIG: UnitGroupConfig = {
+  label: 'Decimal',
+  decimal_places: true,
+  abbreviate: true,
+};
+const PERCENT_GROUP = 'Percent';
+export const PERCENT_UNIT_CONFIG: Readonly<Record<PercentUnitKind, UnitConfig>> = {
+  Percent: {
+    group: PERCENT_GROUP,
+    label: 'Percent (0-100)',
+  },
+  PercentDecimal: {
+    group: PERCENT_GROUP,
+    label: 'Percent (0-1)',
+  },
+  '%': {
+    group: PERCENT_GROUP,
+    // TODO: figure out how this is different
+    label: '% (TBD???)',
+  },
```

Comment:
I'm a little confused about what the purpose of this option is? Is it a shorthand version of `Percent` or something else? Should this be represented in the dropdown? If so, what label should it get to clarify the difference?

Reply:
Yeah this is shorthand like you said and shouldn’t be in the dropdown, forgot it was in there honestly

Reply:
Thanks for the context! I'll add a config option that lets an item be ignored in the dropdown.

Reply:
We could maybe just get rid of it completely, it would be a breaking change but I don’t think anyone is really using it, thoughts @LukeTillman?

Reply:
I'll leave this as is for this PR, and we can potentially do that breaking change in a follow-up if we decide it's a good idea.